### DISCUSSION TITLE
Alexa sparita ?

### DISCUSSION THREAD

**Discussion by riccionese** [indomus/forum#959] [HUB: Home Assistant]
Oggi mi cono accorto che tutte le entità **alexa media player*** sono disabilitate e non capisco come riabilitarle
capitato solo a me ?

**Comment by Scialla** [comment]
A volte basta un riavvio di HA, altrimenti devi guardare i log per capire se c'è un problema con l'OTP e l'autenticazione.

Ti rimando alla guida:
https://indomus.it/guide/integrare-amazon-alexa-come-media-player-su-home-assistant/

**Reply by riccionese** [reply]
Il riavvio l'ho già fatto (anche un reboot) 
Nei log ci ho guardato al volo e non mi sembra di aver visto nulla ma ... ero decisamente distratto da al tre inezie (sono al lavoro  :-) )
ci guarderò meglio

**Reply by Scialla** [reply]
sulla pagina delle integrazioni lo vedi subito se l'ha caricata e, se hai perso tutte le entità, non l'ha caricata..

**Comment by gsarcl366** [comment]
Anche a me è successo: sono sparite le entità. Ad esempio Echo plus ha di suo il sensore di temperatura che usavo in climate. Ora è sparito e non so più come inserirlo tra le entità
